source/Calamari.Tests/KubernetesFixtures/KubernetesRunsScriptFixture.cs

```diff
@@ -365,10 +383,22 @@ CalamariResult ExecuteScriptInternal(ICommandLineRunner runner, IScriptWrapper w
 
         class RecordOnly :  ICommandLineRunner
         {
+            IVariables Variables;
+            public RecordOnly(IVariables variables)
+            {
+                Variables = variables;
+            }
+            
             public CommandResult Execute(CommandLineInvocation invocation)
             {
+                // If were running an aws command (we check the version and get the eks token endpoint) we want to see the actual commands result.
```

Comment:
When checking the aws cli version & get token commands I didn't want to mock a reply. This is a bit hacky in that we have a RecordOnly CommandLineRunner and then create a new CommandLineRunner that records inside it. I'm happy to take another look at this later & any guidance for a refactor would be appreciated.